### PR TITLE
Feature pythoncompiler

### PR DESCRIPTION
**Pull request by JRojas999**

### CODE REVIEW

**Review by jpgonzales27 — APPROVED** [review]

File: src/main/java/org/fundacion/jala/converter/service/pythoncompiler/PythonCompiler.java [hunk 1]
```diff
@@ -6,14 +6,15 @@
...
+ */
 package org.fundacion.jala.converter.service.pythoncompiler;
 
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStreamReader;
+import java.io.*;
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/service/pythoncompiler/PythonCompiler.java:17]: It should call the libraries specifically, not with a *.

File: src/main/java/org/fundacion/jala/converter/service/pythoncompiler/PythonCompiler.java [hunk 2]
```diff
@@ -41,4 +42,32 @@ public String compiler(final Python compilerVersion, final String filePath) {
             return exception.getMessage();
         }
     }
+
+    /**
+     * Converts the code that is written as a string to a file with a python extension
+     * @param code It is a string ready to be converted
+     * @return path where will be created the file
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/service/pythoncompiler/PythonCompiler.java:49]: path where the file will be created is better

File: src/main/java/org/fundacion/jala/converter/service/pythoncompiler/PythonCompiler.java [hunk 3]
```diff
@@ -41,4 +42,32 @@ public String compiler(final Python compilerVersion, final String filePath) {
             return exception.getMessage();
         }
     }
+
+    /**
+     * Converts the code that is written as a string to a file with a python extension
+     * @param code It is a string ready to be converted
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/service/pythoncompiler/PythonCompiler.java:48]: Remove the It. 
code is a String ready to be converted

File: src/main/java/org/fundacion/jala/converter/controller/PythonCompilerController.java [hunk 4]
```diff
@@ -44,9 +44,12 @@ public String uploadFile(final @RequestParam("file") MultipartFile file,
     /**
      * Endpoint for proving compile python
      */
-    @GetMapping("/compilePython2")
-    public String uploadFile2() {
+    @PostMapping("/compilePython2")
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/controller/PythonCompilerController.java:47]: I believe it should have a more representative name. Both the method and the mapping path.

**Review by Raymundogs — APPROVED** [review]

**Pull request merged**